src/Microsoft.Health.SqlServer/Registration/SqlServerBaseRegistrationExtensions.cs

```diff
@@ -95,6 +95,8 @@ public static class SqlServerBaseRegistrationExtensions
                 .AsSelf()
                 .AsImplementedInterfaces();
 
```

Comment:
Not sure where I should place this comment: Should this change include an update to the version number assocated with the Microsoft.Health.SqlServer nuget package?

Reply:
If I understand your ask correctly - Once this PR is merged to the master and published from the release pipeline, we should create a PR to bump the healthcareSharedPackage version in the fhir-server repo.